.github/workflows/unversioned_pages.yml

```diff
@@ -32,11 +32,14 @@ jobs:
           rm -rf ./stable/roadmaps
           cp -Rv ./dev/roadmaps/ ./stable/
           cp -v ./dev/index.html ./stable/
-          find stable/developers -type f -exec sed -i 's+/_static+/../dev/_static+g' {} +
-          find stable/naps -type f -exec sed -i 's+/_static+/../dev/_static+g' {} +
-          find stable/release -type f -exec sed -i 's+/_static+/../dev/_static+g' {} +
-          find stable/roadmaps -type f -exec sed -i 's+/_static+/../dev/_static+g' {} +
+          find stable/developers/ -type f -exec sed -i 's+/_static+/../dev/_static+g' {} +
+          find stable/naps/ -type f -exec sed -i 's+/_static+/../dev/_static+g' {} +
+          find stable/release/ -type f -exec sed -i 's+/_static+/../dev/_static+g' {} +
+          find stable/roadmaps/ -type f -exec sed -i 's+/_static+/../dev/_static+g' {} +
```

Comment:
@melissawm is this a fix for a previous step or just a different notation?

Reply:
I think this is more correct? It's not the priority but I realized this when writing the next step. Happy to revert if we don't want to touch something that's working.